_posts/2025-03-17-collecting-diagnostic-data.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "Collecting diagnostic data"
+date: 2025-03-17
+author: federico_valeri
+---
+
+Debugging distributed systems like Apache Kafka can be challenging.
+Without access to the right diagnostic data, identifying the root cause of an issue can become a time-consuming process.
+Strimzi, the Kubernetes-native Kafka operator, provides a [bash script](https://github.com/strimzi/strimzi-kafka-operator/blob/0.45.0/tools/report.sh) to collect cluster artifacts, including logs and configurations.
+
+Sometimes, this is not enough and we need persistent high-capacity storage to collect custom and advanced diagnostic data.
```

Comment:
shall we explain more about when the default collection is insufficient? When we need more details or longer term retention?